packages/blade/src/components/ButtonGroup/__tests__/ButtonGroup.web.test.tsx

```diff
@@ -168,40 +167,7 @@ describe('<ButtonGroup />', () => {
         </ButtonGroup>,
       ),
     ).toThrowError(
-      '[Blade: ButtonGroup]: Only "Button" or "Dropdown" component with Button trigger are allowed as children.',
-    );
-  });
-
-  it('should throw error with invalid dropdown children', () => {
```

Comment:
why remove this test?

Reply:
Removed this test because useVerifyAllowedChildren already handles the logic, 

plus this was a bit too defensive i felt, previously we used to check for nested components like ButtonGroup->Dropdown->DropdownButton, this is not really needed since, anyways anything other than button is totally visually broken inside ButtonGroup. Not to mention makes composition much harder like you can't put `<Tooltip><DropdownButton /></Tooltip>`